demo/src/plugins/dashboard/components/active-edits/TableRow.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Collaborators from '@triniti/cms/plugins/raven/components/collaborators';
+import Message from '@gdbots/pbj/Message';
+import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';
+import pbjUrl from '@gdbots/pbjx/pbjUrl';
+import { Button, Icon, RouterLink } from '@triniti/admin-ui-plugin/components';
+
+const TableRow = ({ idx, node }) => (
+  <tr key={node.get('_id')}>
+    <td style={{ width: '1px', textAlign: 'right' }}>{idx + 1}.</td>
+    <td>
+      {node.get('title')}
+      <Collaborators nodeRef={NodeRef.fromNode(node)} />
+    </td>
+    <td>
+      {node.get('is_locked') && <Icon imgSrc="locked-solid" alt="locked-status" />}
+    </td>
+    <td>
+      {node.schema().getCurie().getMessage()}
+    </td>
+    <td className="td-icons">
+      <RouterLink to={pbjUrl(node, 'cms')}>
+        <Button id={`view-${node.get('_id')}`} color="hover" radius="circle" className="mb-0 mr-1">
```

Comment:
@nanetteCodes the reason these have ids is because wherever they were copied from had tooltips. please add them here and in the existing dashboard component

Reply:
added these